packages/store/src/secure-store/index.ts

```diff
@@ -1,97 +1,102 @@
-            if (AppMKeytarDirs.length > 0) {
-                // try to load keytar from the first directory found
-                const keytarDir = AppMKeytarDirs[0];
-                log.info('Try to load keytar from :' + JSON.stringify(keytarDir) + '\n');
-                // Support bundling
-                if (typeof __non_webpack_require__ === 'function') {
-                    return __non_webpack_require__(keytarDir);
-                } else {
-                    return require(keytarDir);
+            const fallbackPaths = [...getZoweSdkPaths(false), ...getZoweSdkPaths(true)];
+            log.info(`Discovered fallback directories: ${JSON.stringify(fallbackPaths)}`);
+
+            for (const path of fallbackPaths) {
+                try {
+                    log.info(`Attempting to load Zowe secrets SDK from: ${path}`);
+                    return typeof __non_webpack_require__ === 'function'
+                        ? __non_webpack_require__(path)
+                        : require(path);
+                } catch (fallbackError) {
+                    log.warn(`Failed to load Zowe secrets SDK from ${path}: ${errorString(fallbackError)}`);
                 }
             }
-        } catch (e) {
-            log.warn(errorString(e));
-            log.warn('Could not get keytar from sap-ux-application-modeler-extension node_modules');
-        }
-
-        log.warn(`Could not "require('keytar')". Trying VSCode's copy`);
-        let vscode;
-
-        try {
-            vscode = require('vscode');
-        } catch (e) {
-            log.warn(errorString(e));
-            log.warn('Could not get hold of vscode');
-            return undefined;
-        }
-        try {
-            return require(`${vscode?.env?.appRoot}/node_modules.asar/keytar`);
-        } catch (e) {
-            log.warn(errorString(e));
-            log.warn('Could not get keytar from vscode node_modules.asar');
-        }
-        try {
-            return require(`${vscode?.env?.appRoot}/node_modules/keytar`);
-        } catch (e) {
-            log.warn(errorString(e));
-            log.warn('Could not get keytar from vscode node_modules');
+        } catch (fallbackDiscoveryError) {
+            log.warn(`Error while discovering fallback paths: ${errorString(fallbackDiscoveryError)}`);
         }
 
+        log.warn('Unable to load Zowe secrets SDK from any location.');
         return undefined;
     }
 }
 
 /**
- * Create an instance of a store
+ * Provides an instance of a secure store.
+ *
+ * @param log - Logger instance for logging.
+ * @returns An instance of SecureStore (KeyStoreManager or DummyStore).
  */
 export const getSecureStore = (log: Logger): SecureStore => {
+    // Use a DummyStore in environments where secure storage is disabled
     if (isAppStudio() || process.env.FIORI_TOOLS_DISABLE_SECURE_STORE) {
+        log.info('Secure store disabled, using DummyStore.');
```

Comment:
```suggestion
        log.debug('Secure store disabled, using DummyStore.');
```
This maybe a confusing log on AppStudio.